static/js/containers/CreatePostPage.js

```diff
@@ -26,6 +27,9 @@ import type {
 import type { RestState } from "../flow/restTypes"
 import type { Dispatch } from "redux"
 import type { Match } from "react-router"
+import { setBannerMessage } from "../actions/ui"
+import { clearPostError } from "../actions/post"
+import { anyErrorExcept404 } from "../util/rest"
```

Comment:
can you move these function imports up with the other function imports above? we typically try to sort our imports following the [style guide](https://github.com/mitodl/eslint-config#import-order)